cocos/core/renderer/scene/submodel.ts

```diff
@@ -371,7 +371,13 @@ export class SubModel {
         // update draw info
         const drawInfo = this._subMesh.drawInfo;
         if (this._inputAssembler && drawInfo) {
-            this._inputAssembler.drawInfo.copy(drawInfo);
+            this._inputAssembler.drawInfo.vertexCount = drawInfo.vertexCount;
+            this._inputAssembler.drawInfo.firstVertex = drawInfo.firstVertex;
+            this._inputAssembler.drawInfo.indexCount = drawInfo.indexCount;
+            this._inputAssembler.drawInfo.firstIndex = drawInfo.firstIndex;
+            this._inputAssembler.drawInfo.vertexOffset = drawInfo.vertexOffset;
+            this._inputAssembler.drawInfo.instanceCount = drawInfo.instanceCount;
+            this._inputAssembler.drawInfo.firstInstance = drawInfo.firstInstance;
```

Comment:
drawInfo.copy does the same logic. Why did you remove `copy` invocation and wrote the logic again?

Reply:
`DrawInfo` has been exported by emscripten, ems value cannot be copied from js drawInfo, so copy here